File: Week 1/task5.py
""" Вывод делителей числа N """
while True:
    NUM = int(input())
    if NUM < 1000000:
        break
k = 0
for i in range(1, NUM+1):
    if NUM % i == 0:
        k += 1
        print(i, end=' ')
if k == 2:
    print("\nACHTUNG")

File: HW4/task5.go
// Вывод делителей числа
package main

import "fmt"

func main() {
	var num, k int
	fmt.Scan(&num)
	for i := 1; i < num+1; i++ {
		if num%i == 0 {
			k++
			fmt.Print(i, " ")
		}
	}
	if k == 2 {
		fmt.Print("\nACHTUNG")
	}
}
